com.unity.shadergraph/Editor/Drawing/Blackboard/BlackboardProvider.cs

```diff
@@ -208,7 +208,7 @@ void AddPropertyItems(GenericMenu gm)
             shaderInputTypes.Sort((s1, s2) => {
                 var info1 = Attribute.GetCustomAttribute(s1, typeof(BlackboardInputInfo)) as BlackboardInputInfo;
                 var info2 = Attribute.GetCustomAttribute(s2, typeof(BlackboardInputInfo)) as BlackboardInputInfo;
-            
+
```

Comment:
Revert this file, only whitespace changes are unnecessary

Reply:
Will do 👍